### ISSUE TITLE
un-ascii string should not escape.

### ISSUE THREAD

**Issue by ch-yx** [vberlier/nbtlib#10]
\>\>\> str(nbtlib.tag.String("我"))
'"\\\\\\\\u6211"'

as NBT String only support \\\" and \\\\ escape.

**Comment by vberlier** [comment]
Unicode escaping works as intended. In this case the extra backslashes are there because the python repl outputs a representation of the string to your terminal rather than the string itself:

```py
>>> 我 = String('我')
>>> str(我)
'"\\\\u6211"'
>>> print(repr(str(我)))  # This is the code that the repl executes
'"\\\\u6211"'
```

If you want to display the actual nbt literal you just need to print the string directly:

```py
>>> print(str(我))
"\\u6211"
>>> print(我)  # The `print` function internally calls `str`
"\\u6211"
```

**Comment by ch-yx** [comment]
but,command do not support  escapes like  \uxxxx.

![2019-01-28_13 49 51](https://user-images.githubusercontent.com/45825511/51816943-c21c0100-2303-11e9-93f0-98dfed85b117.png)
🤔

**Comment by vberlier** [comment]
Oh right my mistake I tested it with json text objects so I didn't realize that the reason why the escape sequence worked was because it was interpreted afterwards by the json parsing:

```py
>>> print(Compound({'display': Compound({'Name': String('{"text":"我"}')})}))
{display:{Name:"{\"text\":\"\\u6211\"}"}}
```

So yeah I tried it out in a command and it worked:

```
/give @p minecraft:apple{display:{Name:"{\"text\":\"\\u6211\"}"}}
```

Anyway since bare strings seem to support unicode characters directly I think I'll just get rid of the escaping.

**Comment by vberlier** [comment]
Alright escaping problems should be fixed in 97efa25d76bb40316450f01dfb5254949d18e6a4.

**Comment by vberlier** [comment]
Closing the issue as the fix is now in release 1.3.1.
